src/PotentialCalculation/Painting.jl

```diff
@@ -70,11 +70,8 @@ function paint!(point_types, potential, face::AbstractSurfacePrimitive{T}, geome
                 if in(pt, grid) && in(pt, geometry, csgtol)
                     point_types[i1, i2, i3] = zero(PointType)
                     potential[i1, i2, i3] = pot_value
-                    if isdefined(det.semiconductor.impurity_density_model, :surface_imp_model)
-                        doped_contact_geom = det.semiconductor.impurity_density_model.surface_imp_model.contact_with_lithium_doped
-                        if in_inactive_contact(pt, doped_contact_geom)
-                            point_types[i1, i2, i3] |= inactive_contact_bit
-                        end
+                    if is_inactive_layer_contact
+                        point_types[i1, i2, i3] |= inactive_contact_bit
```

Comment:
Not sure if this makes a performance difference, but one could also do
```julia
point_types[i1, i2, i3] |= inactive_contact_bit * is_inactive_layer_contact
```
and omit the `if`-statement here (also everywhere else).